tasks/collections.yml

```diff
@@ -65,7 +65,7 @@
 
 - name: Solr | Modify existing collections
   uri:
-    url: "{{ solr_url }}/admin/collections?action=MODIFYCOLLECTION&collection={{ item }}&replicationFactor={{ solr_collections[item].replicas }}&maxShardsPerNode={{ solr_collections[item].shards_per_node }}&autoAddReplicas=true"
+    url: "{{ solr_url }}/admin/collections?action=MODIFYCOLLECTION&collection={{ item }}&replicationFactor={{ solr_collections[item].replicas }}&maxShardsPerNode={{ solr_collections[item].shards_per_node }}&autoAddReplicas={{ solr_collections[item].auto_add_replicas | default('true') }}"
```

Comment:
I would prefer to have a default (and overridable) variable to manage that behaviour instead of defaulting to true if there is no value in "solr_collections[item].auto_add_replicas"

Reply:
Done, already extracted **auto_add_replicas** default harcoded config value to a variable that can be overridden (**solr_auto_add_replicas**), at creating and modifying existing collections steps.

Reply:
good job!

IMHO it would be nice to check if this [collections API endpoint ](https://solr.apache.org/guide/6_6/collections-api.html#CollectionsAPI-clusterprop) returns ```autoAddReplicas: "true"``` using molecule

Reply:
Yeah, I already checked in the solr admin GUI while testing the role change (two collections, one with default value "test1" and the other one setting a custom value "test2").
I've checked again but I've not found a "get property value" option in that endpoint... :( I see only edit/add options. 
Anyway I've used this one that allows to check cluster status including property values autoAddReplicas:

(using command line)
test1 collection (var not set, using default value)
```
http "http://localhost:8983/solr/admin/collections?action=CLUSTERSTATUS&wt=json&collection=test1"
HTTP/1.1 200 OK
Content-Length: 2859
Content-Type: application/json;charset=utf-8

{
    "cluster": {
        "collections": {
            "test1": {
                "autoAddReplicas": "True",
                "configName": "test1",

[...]
```
test2 collection (custom value false)
```

http "http://localhost:8983/solr/admin/collections?action=CLUSTERSTATUS&wt=json&collection=test2"
HTTP/1.1 200 OK
Content-Length: 2860
Content-Type: application/json;charset=utf-8

{
    "cluster": {
        "collections": {
            "test2": {
                "autoAddReplicas": "False",
                "configName": "test2",
[...]
```

Reply:
As @dortegau  suggested I will add also a check in molecule to verify the previous comment behaviour

Reply:
I've already added verify step in molecule to check autoAddReplicas is true by default if not defined. I've just had to add a new collection `sample_techproducts_configs_2` in which the var `auto_add_replicas` is not defined.